ChangeLog.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 ### Added
 - More unit tests that cover the `Init` class.
+- Fixed a bug where `null` or empty string request packets would cause an exception to be thrown when calling the `Init` constructor.
```

Comment:
This release log should be under a `### Fixed` subheading. See here for more info: https://keepachangelog.com/en/1.0.0/

Also, this seems to be copy+paste from the PHP SDK. Are we sure that this problem is identical? For example did it fail on empty strings, or did it only fail when `null` was passed?

Reply:
I confirmed by following the code that the error would happen due to an empty string. JsonObjectFactory.fromString turns an empty string into a null, so that would then cause the exception to be thrown.

Reply:
Ah cool. 👍 So it really was pretty much identical to Central lol.